automatic/python3/tools/chocolateyInstall.ps1

```diff
@@ -1,28 +1,51 @@
 ﻿# This file should be identical for all python3* packages
+# https://docs.python.org/3/using/windows.html#installing-without-ui
 
-$packageName = '{{PackageName}}'
-$url = '{{DownloadUrl}}'
-$url64 = '{{DownloadUrlx64}}'
-$version = '{{PackageVersion}}'
-$fileType = 'exe'
-$partialInstallArgs = '/quiet InstallAllUsers=1 PrependPath=1'
+$packageName = 'python3'
+$url32       = 'https://www.python.org/ftp/python/3.5.2/python-3.5.2.exe'
+$url64       = 'https://www.python.org/ftp/python/3.5.2/python-3.5.2-amd64.exe'
+$checksum32  = '529c46b9fd3dcf83029b8bfc95034e640ea2c69835b1aa4b75edeec8de764193'
+$checksum64  = '2cfcdc77a0ba403acf72ba217898fb7c06ce778a5cb85f5220fd32127e40f263'
 
-$installPath = Split-Path -Parent $MyInvocation.MyCommand.Definition
+$installDir   = '{0}\Python{1}' -f $Env:SystemDrive, ($Env:ChocolateyPackageVersion -replace '\.').Substring(0,2)
+$installArgs  = '/quiet InstallAllUsers=1 PrependPath=1 TargetDir="{0}"' -f $installDir
+if ($Env:ChocolateyPackageParameters -match '/InstallDir:\s*(.+)') {
+    $installDir = $Matches[1]
+    if ($installDir.StartsWith("'") -or $installDir.StartsWith('"')){  $installDir = $installDir -replace '^.|.$' }
+    $parent = Split-Path $installDir
+    mkdir -force $parent -ea 0 | out-null
+}
 
-$installArgs = $($partialInstallArgs + ' TargetDir="' + $installPath + '"')
+$params = @{
+  packageName    = $packageName
+  fileType       = 'EXE'
+  silentArgs     = $installArgs
+  url            = $url32
+  url64Bit       = $url64
+  checksum       = $checksum32

```

Comment:
When you `./update.ps1`.

I currently left template stuff because it doesn't matter. After you run update for the firest time package will be valid.